apps/relgen/src/index.ts

```diff
@@ -77,7 +77,7 @@ const cli = program
 
     if (config) {
       configFile = configSchema.parse(
-        JSON.parse(await readFile(config, 'utf-8'))
```

Comment:
The use of `trim` here should be avoided. Please find an alternative method to handle whitespace.